docs/content.zh/docs/concepts/overview.md

```diff
@@ -104,6 +122,7 @@ drwxr-xr-x 2 9999 9999 60 May 12 09:46 a6031ec7-ab3e-4b30-ba77-6498e58e6b7f
 drwxr-xr-x 2 9999 9999 60 May 11 15:11 b6fb2a9c-d1cd-4e65-a9a1-e825c4b47543
 ```
 
-### AuditUtils can log sensitive information present in the custom resources
-As reported in [FLINK-30306](https://issues.apache.org/jira/browse/FLINK-30306) when Flink custom resources change the operator logs the change, which could include sensitive information. We suggest ingesting secrets to Flink containers during runtime to mitigate this.
-Also note that anyone who has access to the custom resources already had access to the potentially sensitive information in question, but folks who only have access to the logs could also see them now. We are planning to introduce redaction rules to AuditUtils to improve this in a later release.
+<a name="auditUtils can log sensitive information present in the custom resources"></a>
```

Comment:
```suggestion
<a name="auditUtils-can-log-sensitive-information-present-in-the-custom-resources"></a>
```

We should keep the `-` style at the link anchors in the whole page.

BTW:I am not sure if it is necessary to lowercase all characters, which needs to be consistent with the English title format. Do you mind checking and making adjustments?

Reply:
done